src/base/settingsstorage.cpp

```diff
@@ -70,9 +70,9 @@ namespace
         SettingsPtr createSettings(const QString &name)
         {
 #ifdef Q_OS_WIN
-            return SettingsPtr(new QSettings(QSettings::IniFormat, QSettings::UserScope, "qBittorrent", name));
+            return SettingsPtr(new QSettings(QSettings::IniFormat, QSettings::UserScope, Utils::Fs::configurationName(), name));
```

Comment:
SettingsStorage is not the only class that requires modifications. There are some other classes that store all their settings/state/data in the same place. Although it may be enough changes in QDesktopServicesDataLocation for them.